indy_common/authorize/auth_request_validator.py

```diff
@@ -33,21 +34,26 @@ def register_default_authorizers(self):
         self.register_authorizer(AndAuthorizer(), auth_constraint_id=AND_CONSTRAINT_ID)
         self.register_authorizer(OrAuthorizer(), auth_constraint_id=OR_CONSTRAINT_ID)
 
-    def validate(self, request: Request, action_list: [AbstractAuthAction], is_owner=False):
+    def validate(self, request: Request, action_list: [AbstractAuthAction]):
         for action in action_list:
             action_id = action.get_action_id()
             auth_constraint = self.auth_cons_strategy.get_auth_constraint(action_id)
             if auth_constraint:
                 try:
                     super().authorize(request=request,
                                       auth_constraint=auth_constraint,
-                                      auth_action=action,
-                                      is_owner=is_owner)
+                                      auth_action=action)
                 except AuthValidationError as exp:
-                    logger.warning("Request {} cannot be authorized by reason: {}".format(request, exp))
-                    return False
+                    logger.warning("Request {} cannot be authorized by reason: {}".format(request, exp.reason))
+                    raise UnauthorizedClientRequest(request.identifier,
+                                                    request.reqId,
+                                                    exp.reason)
                 return True
-            return False
+            error_msg = "There is no authorization constraints for request: {}".format(request)
```

Comment:
It's better to say:
`Request can not be authorized as action is not allowed: {}`.format(request)